charts/windocks/v1.5.0/questions.yml

```diff
@@ -0,0 +1,110 @@
+categories:
+- Database
+- SQL Server
```

Comment:
WIndocks-sql-proxy can run on Linux. The external Windocks server is on Windows, we have a demo VM set up at that IP so that IP works. Users will change it to their VM